class UserNotFoundException(Exception):
    '''User not found exception'''
    pass
